lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -107,6 +107,16 @@ def store(credit_card, options={})
         end
       end
 
+      def unstore(options={})
+        requires!(options, :shopper_reference)
+        post = init_post(options)
+
+        post[:shopperReference] = options[:shopper_reference]
+        post[:merchantAccount] = options[:merchant_account] || @merchant_account
```

Comment:
Good catch -- we don't want `init_post` here. 👍

Reply:
Not a big deal to me, but I'm wondering if we should have methods for `add_shopper_reference` and `add_merchant_account` since we are setting those here, in `init_post`, and in `add_extra_data`.